python/requirements.txt

```diff
@@ -312,7 +312,9 @@ pure-eval==0.2.2
 py-partiql-parser==0.4.2
     # via moto
 pyarrow==14.0.1
```

Comment:
streamlit has a pyarrow dependency? Very interesting

Reply:
Still good to explicitly add it to requirements.in, just surprised to already see it here